crates/fluvio-connector-package/src/config.rs

```diff
@@ -223,6 +223,21 @@ impl From<BTreeMap<String, String>> for ManagedConnectorParameterValue {
     }
 }
 
+impl ManagedConnectorParameterValue {
```

Comment:
Let's change `ManagedConnectorParameterValue` to `ConnectorParameterValue` since CDK API will be general purpose

Reply:
done